src/modules/collection/utils.ts

```diff
@@ -38,10 +39,13 @@ export function getCollectionEditorURL(collection: Collection, items: Item[]): s
   return locations.itemEditor({ collectionId: collection.id, itemId: items.length > 0 ? items[0].id : undefined })
 }
 
-export function getExplorerURL(collection: Collection) {
+export function getExplorerURL(collection: Collection, itemStatus?: CurationStatus) {
   const EXPLORER_URL = env.get('REACT_APP_EXPLORER_URL', '')
   const BUILDER_SERVER_URL = env.get('REACT_APP_BUILDER_SERVER_URL', '')
-  return `${EXPLORER_URL}?WITH_COLLECTIONS=${collection.id}&BUILDER_SERVER_URL=${BUILDER_SERVER_URL}&NETWORK=ropsten&DEBUG_MODE=true`
+  return (
+    `${EXPLORER_URL}?WITH_COLLECTIONS=${collection.id}&BUILDER_SERVER_URL=${BUILDER_SERVER_URL}&NETWORK=ropsten&DEBUG_MODE=true` +
```

Comment:
With so many query params wouldn't it be nice to use the native `URLSearchParams`?